language/pl-PL/com_users.ini

```
@@ -4,119 +4,159 @@
+COM_USERS_RESET_REQUEST_ERROR="Wystąpił błąd podczas resetowania hasła."
 COM_USERS_RESET_REQUEST_FAILED="Niepomyślna zmiana hasła: %s"
-COM_USERS_RESET_REQUEST_LABEL="Nie pamiętasz hasła? To nie jest problem. Wpisz poniżej swój adres poczty elektronicznej podany podczas rejestracji i kliknij przycisk &bdquo;Wyślij&rdquo;. Otrzymasz specjalny kod umożliwiający zalogowanie się i dokonanie zmiany hasła.<br />Sprawdź za kilka chwil swoją skrzynkę pocztową. Gdy otrzymasz wiadomość z kodem weryfikującym, skopiuj go i podaj podczas logowania, a następnie dokonaj zmiany hasła."
+COM_USERS_RESET_REQUEST_LABEL="Wprowadź adres e-mail swojego konta. Kod weryfikacyjny zostanie wysłany do Ciebie. Po otrzymaniu kodu weryfikacyjnego będziesz mógł wybrać nowe hasło do swojego konta."
 COM_USERS_SETTINGS_FIELDSET_LABEL="Pełna nazwa:"
 COM_USERS_USER_BACKUPCODE="Kod odzyskiwania"
 COM_USERS_USER_BACKUPCODES="Kody odzyskiwania"
-COM_USERS_USER_BACKUPCODES_CAPTIVE_PROMPT="Jeśli nie masz dostępu do swojej zwykłej metody uwierzytelniania wieloskładnikowego, użyj dowolnego kodu zapasowego w polu poniżej. Pamiętaj, że ten awaryjny kod zapasowy nie może być ponownie użyty."
-COM_USERS_USER_BACKUPCODES_DESC="Jeśli nie masz dostępu do urządzenia uwierzytelniania wieloskłądnikowego, możesz użyć dowolnego z poniższych haseł zamiast zwykłego kodu bezpieczeństwa. Każdy z tych kodów awaryjnych jest natychmiast niszczony po użyciu. Zalecamy wydrukowanie tych kodów i przechowywanie wydruku w bezpiecznym i dostępnym miejscu, np. w portfelu lub skrytce depozytowej."
-COM_USERS_USER_BLOCKED="Ten użytkownik jest zablokowany. Jeśli jest błąd, prosimy o kontakt z administratorem."
-COM_USERS_USER_FIELD_BACKEND_LANGUAGE_LABEL="Wybierz styl szablonu zaplecza administracyjnego. Dotyczy tylko tego użytkownika."
-COM_USERS_USER_FIELD_BACKEND_TEMPLATE_LABEL="Edytor użytkownika"
+COM_USERS_USER_BACKUPCODES_CAPTIVE_PROMPT="Jeśli nie masz dostępu do zwykłej metody uwierzytelniania wieloskładnikowego, użyj któregokolwiek ze swoich kodów zapasowych w polu poniżej. Pamiętaj, że ten awaryjny kod zapasowy nie może być ponownie użyty."
+COM_USERS_USER_BACKUPCODES_DESC="Jeśli nie masz dostępu do urządzenia uwierzytelniania wieloskłądnikowego, możesz użyć dowolnego z poniższych haseł zamiast zwykłego kodu bezpieczeństwa. Każdy z tych kodów awaryjnych jest natychmiast niszczony po użyciu. Zalecamy wydrukowanie tych kodów i przechowywanie wydruku w bezpiecznym i dostępnym miejscu, np. w portfelu lub sejfie"
+COM_USERS_USER_BLOCKED="Ten użytkownik jest zablokowany. Jeśli jest to błąd, skontaktuj się z administratorem."
+COM_USERS_USER_FIELD_BACKEND_LANGUAGE_LABEL="Język zaplecza"
+COM_USERS_USER_FIELD_BACKEND_TEMPLATE_LABEL="Styl szablonu zaplecza"
 COM_USERS_USER_FIELD_EDITOR_LABEL="Wpisz adres e-mail tego użytkownika."
-COM_USERS_USER_FIELD_FRONTEND_LANGUAGE_LABEL="Wybierz język witryny. Dotyczy tylko tego użytkownika."
+COM_USERS_USER_FIELD_FRONTEND_LANGUAGE_LABEL="Język interfejsu"
```

Fronton nazywamy witryną. Też od lat.  Zatem: Język witryny
```suggestion
COM_USERS_USER_FIELD_FRONTEND_LANGUAGE_LABEL="Język witryny"
```